Bugzilla.pm

```diff
@@ -767,6 +768,23 @@ sub elastic {
     $class->process_cache->{elastic} //= Bugzilla::Elastic->new();
 }
 
+sub check_rate_limit {
+    my ($class, $name, $id) = @_;
+    my $params = Bugzilla->params;
+    if ($params->{rate_limit_active}) {
```

Comment:
Moving this specific test up one level to the function calling ```check_rate_limit``` would improve the performance of rate limiting checks, very very slightly.

Normally I wouldn't suggest micro-optimizations, but since this function is used in the anonymous user request paths, if there's heavy load, any improvement may be worth considering.

If you choose to pursue this, you'd need a comment indicating that the caller is responsible for that test.